### ISSUE TITLE
[Docs] Clarify Subphrase classification?

### ISSUE THREAD

**Issue by hamelsmu** [huggingface/transformers#11482]
I am going through the docs linearly and am reading [Summary of The tasks](https://huggingface.co/transformers/task_summary.html).

The second section of [Sequence Classification](https://huggingface.co/transformers/task_summary.html) uses `bert-base-cased-finetuned-mrpc` to do paraphrase classification.  

This is a bit opaque to me, as when I go to the [model page](https://huggingface.co/bert-base-cased-finetuned-mrpc) for that particular model, it doesn't really mention this capability?

How could I discover other models that have this capability?  How do I verify what this model was fine-tuned on if I was searching for this information from the model hub?  Is there some other documentation about this that I am missing? 

Just trying to understand so I can help clarify the docs.  Thanks!

@sgugger

**Comment by sgugger** [comment]
The problem lies mostly on the model page in that case, it should show the text classification widget, not the masked LM widget. As for adding a model card or expanding its capabilities, the feedback should go on the [forums](https://discuss.huggingface.co/), this is not really an issue with transformers per se.

This is a great example of where pull requests on the model hub would be useful! (cc @julien-c )

**Comment by hamelsmu** [comment]
Ok thanks I'll close this issue and open an appropriate PR/Issue in other places.  I'll try to find where to update the filter and I'll put the model card on my todo list.  

Thanks for the pointers

**Comment by hamelsmu** [comment]
For anyone that finds this issue:

 - [here is the forum post that describes how to suggest model cards](https://discuss.huggingface.co/t/about-the-model-cards-category/2777)

- For clarification `bert-base-cased-finetuned-mrpc` does indeed do masked LM but it additionally also does text classification (I tried doing both)

**Comment by julien-c** [comment]
- @hamelsmu The model card for this model should a minima reference the `mrpc` dataset, though we don't have it as a standalone dataset so the way to go would be to link to `glue` instead. (right @lhoestq?)
- you can also add a `tags: - paraphrase-classification ` to the YAML (tags are pretty much open)
- read the doc about the model hub here http://huggingface.co/docs (should it be linked more prominently from the transformers doc?)
- As this is a "legacy" model (not inside an organization), it's hard to remember who trained it and therefore could answer more questions (the original BERT authors? Someone from HF?)
- To your last question, most models only have one head so one task – for simplicity we enforce this constraint of only having one widget or Inference API endpoint per model

**Comment by lhoestq** [comment]
Yes for now we have to link to `glue`.
Though I've noticed that many models use the `mrpc` tag that doesn't link to glue

Maybe we can define a syntax that mentions both glue (to link to the glue dataset page) and MRPC (to mention which config if the glue dataset was used). Maybe `glue/mrpc` or `glue:mrpc`. Shall I open an issue on the website repo about this @julien-c ?
